pages/linux/resolvectl.md

```diff
@@ -0,0 +1,33 @@
+# resolvectl
+
+> Resolve domain names, IPV4 and IPv6 addresses, DNS resource records, and services.
+> Introspect and reconfigure the DNS resolver.
+> More information: <https://www.freedesktop.org/software/systemd/man/resolvectl.html>.
+
+- Show DNS settings:
+
+`resolvectl status`
+
+- Resolve the IPv4 and IPv6 addresses for one or more domains:
+
+`resolvectl query {{domain1 domain2 ...}}`
+
+- Retrieve the domain of a specified IP:
+
+`resolvectl query {{ip_address}}`
+
+- Retrieve an MX record of domain:
+
+`resolvectl  --legend=no -t MX query  {{domain}}`
```

Comment:
We prefer long arguments, and there is extra spaces.
```suggestion
`resolvectl --legend={{no}} --type={{MX}} query {{domain}}`
```

Reply:
I selected commit suggestion, yet it still says 1 change requested. Is there anything else that needs to be changed?